File: Lab03/Python_Lang/Exercise03/include/array.py
#
#  Created by NDHuy on 21/05/2020
#  Copyright 2020 by SEstudio
#  Array 01 Dimension
#

def Enter(a):
    n = int(input("Enter num of array: "))
    i = 0
    while(i < n):
        temp = int(input("a[%d] = " %(i)))
        a.append(temp)
        i += 1


def Output(a):
    s = ""
    for i in a:
        s += str(i) + "\t"
    print(s)


def PrintOdds(a):
    s = "Odds item: "
    for i in a:
        if(i % 2 != 0):
            s += str(i) + "\t"
    print(s)


def PrintEvens(a):
    s = "Evens item: "
    for i in a:
        if(i % 2 == 0):
            s += str(i) + "\t"
    print(s)


def CountValue(a, value):
    count = 0
    for i in a:
        if(i == value):
            count += 1

    return count

File: 01_BasicPagramming/Lab03/C_Lang/Exercise03/src/array.cpp
//
//  Created by NDHuy on 21/05/2020
//  Copyright 2020 by SEstudio
//  Array 01 Dimension
//

#include "array.h"

void Input(int a[], int &n)
{
    do
    {
        printf("Enter num of array n = ");
        scanf("%d", &n);

        if(0 > n || n > MAX)
            printf("n must between 0 and %d. Try again later!\n", MAX);
    } while (0 > n || n > MAX);

    int temp;
    for(int i = 0; i < n; i++)
    {
        printf("a[%d] = ", i);
        scanf("%d", &temp);
        a[i] = temp;
    }
}

void Output(const int a[], const int &n)
{
    for(int i = 0; i < n; i++)
    {
        printf("%d\t", a[i]);
    }
    printf("\n");
}

void PrintOdds(const int a[], const int &n)
{
    printf("Odds item: ");
    for(int i = 0; i < n; i++)
    {
        if(a[i] % 2 != 0)
            printf("%d\t", a[i]);
    }
    printf("\n");
}

void PrintEvens(const int a[], const int &n)
{
    printf("Evens item: ");
    for(int i = 0; i < n; i++)
    {
        if(a[i] % 2 == 0)
            printf("%d\t", a[i]);
    }
    printf("\n");
}

int CountValue(const int a[], const int &n, const int value)
{
    int count = 0; 
    for(int i = 0; i < n; i++)
    {
        if(a[i] == value)
            count++;
    }
    return count;
}